shared/data/tracker_lists/trackersWithParentCompany.json

```diff
@@ -7190,269 +7190,74 @@
-            "c": "Yahoo!",
-            "u": "http://www.yahoo.com/"
-        },
-        "buzz.yahoo.com": {
-            "c": "Yahoo!",
-            "u": "http://www.yahoo.com/"
-        },
-        "calendar.yahoo.com": {
-            "c": "Yahoo!",
-            "u": "http://www.yahoo.com/"
-        },
-        "edit.yahoo.com": {
-            "c": "Yahoo!",
-            "u": "http://www.yahoo.com/"
-        },
-        "legalredirect.yahoo.com": {
-            "c": "Yahoo!",
-            "u": "http://www.yahoo.com/"
-        },
-        "login.yahoo.com": {
-            "c": "Yahoo!",
-            "u": "http://www.yahoo.com/"
-        },
-        "mail.yahoo.com": {
-            "c": "Yahoo!",
-            "u": "http://www.yahoo.com/"
-        },
-        "my.yahoo.com": {
-            "c": "Yahoo!",
-            "u": "http://www.yahoo.com/"
-        },
-        "notepad.yahoo.com": {
-            "c": "Yahoo!",
-            "u": "http://www.yahoo.com/"
-        },
-        "pulse.yahoo.com": {
-            "c": "Yahoo!",
-            "u": "http://www.yahoo.com/"
-        },
-        "webmessenger.yahoo.com": {
-            "c": "Yahoo!",
-            "u": "http://www.yahoo.com/"
-        },
-        "ymail.com": {
-            "c": "Yahoo!",
-            "u": "http://www.yahoo.com/"
-        },
-        "reddit.com": {
-            "c": "reddit",
-            "u": "http://www.reddit.com/"
-        },
         "facebook.com": {
             "c": "Facebook",
-            "u": "http://www.facebook.com/"
-        },
-        "facebook.de": {
-            "c": "Facebook",
-            "u": "http://www.facebook.com/"
-        },
-        "facebook.fr": {
-            "c": "Facebook",
-            "u": "http://www.facebook.com/"
+            "u": "https://facebook.com",
+            "rules": [
+                {"rule": "facebook\\.com\\/ai\\.php\\?"},
+                {"rule": "facebook\\.com\\/audience_network\\/", "options": {"types": ["image"]}},
+                {"rule": "pixel\\.facebook\\.com($|[?/])"},
+                {"rule": "facebook\\.com.*\\/impression\\.php"},
+                {"rule": "graph\\.facebook\\.com\\/\\?ids=.*&callback=.*", "options": {"types": ["script"]}},
+                {"rule": "facebook\\.com\\/.*\\/plugins\\/send_to_messenger\\.php\\?app_id="},
+                {"rule": "facebook\\.com\\/brandlift\\.php"},
+                {"rule": "facebook\\.com\\/tr([?/])"},
+                {"rule": "facebook\\.com\\/common\\/scribe_endpoint\\.php"},
+                {"rule": "facebook\\.com\\/email_open_loc_pic\\.php"},
+                {"rule": "facebook\\.com\\/fr\\/u\\.php\\?"},
+                {"rule": "facebook\\.com\\/method\\/links\\.getStats\\?"},
+                {"rule": "facebook\\.com\\/offsite_event\\.php"},
+                {"rule": "facebook\\.com\\/rtb_impression\\/\\?"},
+                {"rule": "facebook\\.com\\/rtb_video\\/\\?"},
+                {"rule": "facebook\\.com([?/]).*\\/tracking\\.js"},
```

Comment:
See the group here, do we need that?  We're not using the group for anything are we?  

So could this become the following?

`facebook\\.com[?/].*\\/tracking\\.js`

Likewise for any other groups in the rules?

Reply:
👍